README.md

```diff
@@ -1,12 +1,75 @@
 # SQL Project
 
 The purpose of this project is to use SQL as an exploratory tool, towards this end we will create a database with it's 
-corresponding tables and import datasets from [ourworldindata.org](www.ourworldindata.org). Finally we'll analyse this 
+corresponding tables and import datasets from [ourworldindata.org](http://www.ourworldindata.org). Finally we'll analyse this 
 information using SQL.
 
-This project was developed following the steps detailed below.
+This project is divided in two parts:
 
-## 1 - Identify the data requirements
+1. Creation of the Database and tables. Data population in the tables.
+    * DB_creation_and_loading/creation_database.py
+    * DB_creation_and_loading\populate_tables.py
+2. Queries on this database.
+    * Data_exploration\data_exploration_caloric_supply.ipynb
+    * Data_exploration\data_exploration_food_expenditure.ipynb
+    * Data_exploration\data_exploration_gdp.ipynb
+    * Data_exploration\data_exploration_protein_consumption_part1.ipynb
+    * Data_exploration\data_exploration_protein_consumption_part2.ipynb
+
+## Using this Notebook
+
+You can use this notebook in the following way:  
+
+1. Use github as a viewer to observe the sql queries with ```.ipynb``` extension. You don't need to have Jupyter Notebook 
+installed. To proceed with this option just click on the files in the directory ```DATA EXPLORATION```.   
+
+2. Clone this repository to create the database in your local machine. You will need to have Python and Jupyter installed
```

Comment:
Second step should be: `Run the jupyter notebooks on the local machine`, steps to achieve this would be: 
- Clone this repository
- Create the database and tables
- Populate the tables
- Run the sql queries